salt/modules/file.py

```diff
@@ -3173,18 +3173,39 @@ def patch(originalfile, patchfile, options="", dry_run=False):
     options
         Options to pass to patch.
 
+    .. note::
+        Windows now supports this as of 3004. In order to use this
+        function in Windows, select the "Use Git and optional Unix
+        tools from the Command Prompt" option when installing Git.
+        This will put all the Unix commands in your system Path to
+        allow patch to be used.
+
+        If you already have git-for-windows installed, check if
+        Git\\usr\\bin\\ is in your system path. The patch.exe binary
+        is typically found within this folder from Git.
+
+
     CLI Example:
 
     .. code-block:: bash
 
         salt '*' file.patch /opt/file.txt /tmp/file.txt.patch
+
+        salt * file.patch C:\\file1.txt C:\\file3.patch
```

Comment:
```suggestion
        salt '*' file.patch C:\\file1.txt C:\\file3.patch
```

Reply:
I can port this over + make the changelog. As for the tests, we don't install git-for-windows in the ci/cd system and patch is not available through pip or any windows package manager (afaik). It's something completely optional and must be installed manually from a user...